src/functions.c

```diff
@@ -370,6 +370,34 @@ void fcallCommandReadOnly(client *c) {
     fcallCommandGeneric(c, 1);
 }
 
+void functionsFlushCommand(client *c) {
+    if (c->argc > 3) {
+        addReplyErrorFormat(c,"wrong number of arguments for '%s' command or subcommand", c->cmd->name);
```

Comment:
"wrong number of arguments for 'flush' subcommand" ? There isn't really a standard here, but it seems weird to give an error about 'function'.

Reply:
I thought it make sense to match the error to the general one here: https://github.com/redis/redis/blob/c7dc17fc0f9341f61be1a1318468409249310316/src/server.c#L5281

Reply:
Many commands use `addReplySubcommandSyntaxError(c)` for this.

Reply:
Thanks @zuiderkwast, fixed.